### PR TITLE
Amending Contributor guidelines

### PR DESCRIPTION
**Pull request by jason-fox**
Based on the consensus of the meeting in Malaga, some MUST requirements have been clarified, amended or reduced in scope to a SHOULD or a MAY.

This PR updates the list of guidelines to align with the actual requirements which will be enforced going forward.

### CODE REVIEW

File: docs/project_badges.md [hunk 1]
```diff
@@ -89,11 +96,10 @@ Should you have any doubt please check
...
 If a project's documentation uses the standard FIWARE CSS, the repository's
-`docs/index.md` - being the opening page of the documentation, **MUST** contain
-equivalent linked badges in the following order:
-
--   **FIWARE Chapter** (Link to https://www.fiware.org/developers/catalogue/ )
--   **Support** (pointer to the support channel)
+`docs/index.md` - being the opening page of the documentation, it **MUST**
+contain equivalent a linked **FIWARE Chapter** badge (with a link to
+https://www.fiware.org/developers/catalogue/ ), and **SHOULD** contain a
+**Support** badge (with a linkp to the support channel)
```
Comment (fgalan) [inline docs/project_badges.md]: Typo: linkp -> link

File: docs/GE_Checklist.md [hunk 2]
```diff
@@ -149,8 +149,8 @@ for new and incubated enablers listed above **MUST** be met.
 
 -   A Stack Overflow tag for the component is registered (Ask FF Staff)
 -   Where it is suitable to provide a global instance of a service, a public
-    instance is deployed in the FIWARE Lab and maintained by the FF team (Ask FF
-    Staff)
+    instance may be deployed in the FIWARE Lab and maintained by the FF team
```
Comment (jmcanterafonseca) [inline docs/GE_Checklist.md]: MAY? (capitalized)

Comment (jason-fox) [reply docs/GE_Checklist.md]: NTC - there is a reason I didn't captialize the MAY there in the text - it isn't a MAY requirement. its more "if this makes sense to do then do it" - it **may** or **may not** be necessary, but you won't lose points if you don't

File: docs/project_badges.md (hunk 1 above)
Comment (jason-fox) [reply docs/project_badges.md]: Fixed - : 9d55e09

**Pull request merged**